qsc/qsc.py

```diff
@@ -17,13 +17,18 @@ class Qsc():
     """
     
     # Import methods that are defined in separate files:
-    from .init_axis import init_axis
+    from .init_axis import init_axis, R0_func, Z0_func, convert_to_spline, \
+        normal_R_spline, normal_phi_spline, normal_z_spline, \
+        binormal_R_spline, binormal_phi_spline, binormal_z_spline, \
+        tangent_R_spline, tangent_phi_spline, tangent_z_spline
```

Comment:
I think we can find a more elegant solution than importing all these functions. My first thought would be to just assign everything from `R0_func` onwards to `self` at the end of `init_axis()`, as is already done for `curvature`, `tangent_cylindrical`, etc. For `R0_func` you could say `self.R0_func = lambda phi: return...`. After changing `convert_to_spline` as suggested above to return the spline object instead of a number, the end of `init_axis` could have a series of lines like `self.normal_R_spline = self.convert_to_spline(normal_cylindrical[:,0])`